spyder/plugins/maininterpreter/confpage.py

```diff
@@ -158,9 +159,48 @@ def setup_page(self):
         umr_layout.addWidget(umr_namelist_btn)
         umr_group.setLayout(umr_layout)
 
+        # Path anaconda executor
+        conda_group = QGroupBox(_("Path Conda executor"))
+        conda_label = QLabel(
+            _(
+                "Select the conda executor path"
+            )
+        )
+        conda_label.setWordWrap(True)
+
+        conda_layout = QVBoxLayout()
+        conda_layout.addWidget(conda_label)
+
+        self.cus_conda_check = self.create_checkbox(
+            _("Use custom conda executor"), "customConda"
+        )
+
+        conda_layout.addWidget(self.cus_conda_check)
```

Comment:
```suggestion
        custom_conda_check = self.create_checkbox(
            _("Set the path to a custom Conda, Mamba or Micromamba executable"),
            "custom_conda"
        )
        conda_layout.addWidget(custom_conda_check)
```

- No need for the checkbox to be a class attribute.
- Improve its text.
- Change option name to go with my suggestion above.